docs/_articles/en/codebuilder/cb-setup.md

```diff
@@ -40,14 +40,7 @@ From Setup, enter `Installed Packages` in the Quick Find box, then select **Inst
 
 ## Add Team Members as Users in the Code Builder Org
 
-Add any team members who aren’t already users in the Code Builder org. For each team member, specify the appropriate license and profile based on their role.
-
-The listed license and profile is required to use Code Builder.
-
-| **Role**    | **License**                                                                                                                                                                                                                 | **Profile**       |
-| ----------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
-| Team Member | [Free Limited Access](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/dev_hub_license.htm) or [Identity](https://help.salesforce.com/s/articleView?id=sf.users_license_types_available.htm&type=5) | DX Limited Access |
-|             |                                                                                                                                                                                                                             |                   |
+Add any team members who aren’t already users in the Code Builder org. 
```

Comment:
@sbudhirajadoc - Thank you for the quick change.  What license can be used has been a common question.  (I understand it was a common question for DevOps Center also.) I suggest we keep the info that Code Builder works with free licenses.  Maybe add a sentence after "Add any team members who aren’t already users in the Code Builder org." - along the lines of:

Code Builder works with the [Identity](https://help.salesforce.com/s/articleView?id=sf.users_license_types_available.htm&type=5) or the [Free Limited Access](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/dev_hub_license.htm) license.

Reply:
done.